daemon/api.go

```diff
@@ -770,29 +769,35 @@ func sideloadSnap(c *Command, r *http.Request) Response {
 
 	// form.File is a map of arrays of *FileHeader things
 	// we just allow one (for now at least)
```

Comment:
This comment can be dropped after the tweaks below. We're now being more clear about what we take.